hadoop-hdds/client/src/test/java/org/apache/hadoop/ozone/client/io/TestECBlockInputStream.java

```diff
@@ -326,7 +326,8 @@ public void testSeekToLength() throws IOException {
         ECStreamTestUtil.createKeyInfo(repConfig, 5, 100);
     try (ECBlockInputStream ecb = new ECBlockInputStream(repConfig,
         keyInfo, true, null, null, streamFactory)) {
-      assertThrows(EOFException.class, () -> ecb.seek(100));
```

Comment:
Same as above, should we check for the exception at `ecb.seek(100 + 1)` rather than removing the check completely?